test/test.md

```diff
@@ -0,0 +1,32 @@
+# Hello World
+
+* this
+* is 
+* a
+* test
+
+> Cool
+
+`Does it read code though?`
+
+[google](http://www.google.com)
+
+![hi](http://freeangrybirdsgame.org/images/bird-what2.png)
+
+*Bolded*
+**Italics**
+_underlines_
+
+***
+
+1. Numbered
+  * sub-list
+2. List
+3. Works
+
+---
+
+    Code through indent
+
+___
```

Comment:
interesting; never realized macOS has these smart substitutions in text editors too haha